tests/AccountTest.py

```diff
@@ -1,37 +0,0 @@
-from LTOCli.Commands import account
```

Comment:
Why no tests anymore? (goes for other deleted files as well)

Reply:
those tests were useless. Need to create new ones, and they should be e2e.

Reply:
there are no other deleted files except the tests

Reply:
Okay.